### PR TITLE
docs: add Before run Shipjs section

### PR DESCRIPTION
**Pull request by tyankatsu0105**
re #414

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 8.16.0
```
Comment (Haroenv) [inline .tool-versions]: What is this required for?

Comment (tyankatsu0105) [reply .tool-versions]: @Haroenv 
Oh sorry...
I use [asdf](https://github.com/asdf-vm/asdf).
Should I remove that?

Comment (Haroenv) [reply .tool-versions]: I'd say yes, because https://github.com/algolia/shipjs/blob/master/.nvmrc is already set :)

Comment (tyankatsu0105) [reply .tool-versions]: Done!

**Comment by eunjae-lee** [comment]
@tyankatsu0105 Thanks for the contribution 🎉 
Feel free to open a new PR to add yourself as a contributor.

```
yarn contributors:add tyankatsu0105 doc
```

**Pull request merged**